scripts/v.rast.stats/v.rast.stats.html

```diff
@@ -31,8 +31,10 @@ <h2>NOTES</h2>
 will be chopped off.
 <p>If a MASK is present, it will be restored after the script finished.
 The script changes temporarily to the resolution of the given raster map.
-<p>If some vectors overlap, only one category will be kept during the
-rasterization process. Statistics for these vectors will thus be partial.
+<p>If an area has several categories in the selected layer (equivalent
+to overlapping polygons in Simple Features), only one category will be
+kept during the rasterization process. Statistics for the skipped
```

Comment:
Shouldn't skipped there be replaced by kept? The skipped will be skipped, no? So the resulting stats for kept categories will be partial.

Reply:
In the case of two polygons partially overlapping, you have three topological areas where one area has two categories, shared with the other two areas:

- area 1: category 1
- area 2: category 2
- area 3: categories 1, 2

Let's assume, for area 3 category 1 has been used: statistics for category 1 are complete and for category 2 incomplete because category 2 has been skipped for area 3. Thus resulting stats for skipped categories are partial.

Reply:
Thanks for clarification, Markus. I re-created the plot in a paper here. So, you mean it's partial because category 2 will be considered only in area 2 in this case then?

Reply:
I have added a more detailed explanation. OK, @veroandreo?

Reply:
Yes, thanks Markus :)